go/vt/vttablet/tabletserver/gc/tablegc.go

```diff
@@ -654,8 +632,7 @@ func (collector *TableGC) Status() *Status {
 		Keyspace: collector.keyspace,
 		Shard:    collector.shard,
 
-		isPrimary: (atomic.LoadInt64(&collector.isPrimary) > 0),
-		IsOpen:    (atomic.LoadInt64(&collector.isOpen) > 0),
+		IsOpen: (atomic.LoadInt64(&collector.isOpen) > 0),
```

Comment:
Why LoadInt64 here whereas we have removed atomic from other places where we are reading  isOpen ? do we not need synchronization anywhere else ?

Reply:
That's because `isOpen` is only changed in `Open()` and `Close()`, which are both using `stateMutex.Lock()` to exclude each other. So those two functions don't need to `LoadInt64`. But `Status()` is not protected with that same mutex. The tradeoff is that it must read `isOpen` with `LoadInt64` because it might run at the same time `Open()` or `Close()` are running.
An alternative is to add `stateMutex.Lock()` to `Status()`, but I prefer not to.